Docs 4254 13.0 u3 release notes (#1571)
truenas/documentation: content/CORE/COREReleaseNotes.md

|---------|------------|----------------|
| 13.0-U3 | Release | To be determined |

## 13.0-U3

November 1, 2022

iXsystems is pleased to announce the release of TrueNAS 13.0-U3.

> This is a maintenance release with some improvements for ACLs and rsync, updates Samba to 4.15.10 and updates the Asigra plugin. It adds the R50m to the Enclosure function and Storj as a Cloud sync service.

Reviewer: ```suggestion
This is a maintenance release with some improvements for ACLs and rsync, updates Samba to 4.15.10 and updates the Asigra plugin. It adds Enclosure Management integration for the 3rd generation R50 and Storj as a new Cloud Sync provider.
```

## TrueNAS 13.0-U3 Changelog

### Improvement

> [NAS-118437](https://ixsystems.atlassian.net/browse/NAS-118437) Update net/samba to Samba 4.15.10

Reviewer: Changelog entries need to be formatted as an unordered list before merge:

```suggestion
* [NAS-118437](https://ixsystems.atlassian.net/browse/NAS-118437) Update net/samba to Samba 4.15.10
```

| Seen In | Key | Summary | Workaround | Resolved In |
|---------|-----|---------|------------|-------------|

> | 13.0-U3 | [NAS-118832] | UI Does not show the correct status on HA systems | There is a known UI caching issue that impacts the status of failover in HA systems. The workaround is to refresh the browser screen or clear the cache after failing-over or making any UI change to update the UI screens to show the correct status of the two nodes. Note, this might require logging into the system again if your token has expired. | Targeted 13.0-U4 |

Reviewer: ```suggestion
| 13.0-U3 | [NAS-118832](https://ixsystems.atlassian.net/browse/NAS-118832) | UI Does not show the correct status on HA systems | There is a known UI caching issue that impacts the status of failover in HA systems. The workaround is to refresh the browser screen or clear the cache after failing-over or making any UI change to update the UI screens to show the correct status of the two nodes. Note, this might require logging into the system again if your token has expired. | Targeted 13.0-U4 |
```
